### ISSUE TITLE
Piwik 2.16.1 no location_provider in piwik_log_visit

### ISSUE THREAD

**Issue by falkh** [ThaDafinser/Piwik-IntranetGeoIP#21]
Hi,
I installed piwik yesterday and then this plugin and couldn't get it to work.
Then I saw in the apache error log this message.
Error in Piwik (tracker): Error query: SQLSTATE[42S22]: Column not found: 1054 Unknown column 'location_provider' in 'field list' In query: INSERT INTO piwik_log_visit (...

I also have just:
Visitor -> Realtime visitor map
and this one is not in the dashboard list:
Visitor -> Location and provider

can't I see the information on the normal visitor map?
regards Falk

**Comment by ThaDafinser** [comment]
That column is part of the piwik built in plugins.
So if you properly installed/activated the GeoIP plugins it should work

**Comment by falkh** [comment]
No it is not there, I already installed GeoIP PHP vers. and aktivated it but this column is not in the DB.
location_citty, location_country,location_ip, location_region are all there but not location_provider.

**Comment by ThaDafinser** [comment]
@falkh you need the provider plugin for it, sorry. 
Alternative: just remove the field from the config (every data field in the config is optional)
https://github.com/piwik/piwik/blob/25b2da923b2dbe999e4b716a37cfa4f804fa1770/plugins/Provider/Provider.php

**Comment by falkh** [comment]
ah ok thank you now it is worken when I activated the Provider Plugin.

**Comment by pfleckenbw** [comment]
Maybe it would be an idea writing down this point in the documentation of the IntranetGeoIP.data.php

**Comment by ThaDafinser** [comment]
@pfleckenbw good idea. Can you write a PR for it? The file is this one https://github.com/ThaDafinser/Piwik-IntranetGeoIP/blob/master/data.example.php
